requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy==1.20.3
+scikit-learn==0.24.2
+matplotlib==3.4.2
+pandas==1.2.4
```

Comment:
pandas is not needed, I guess...

Reply:
As mentioned, I have seen it being used in `load_data.py`